www-client/luakit/luakit-9999.ebuild

```diff
@@ -0,0 +1,64 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit toolchain-funcs
+
+DESCRIPTION="A fast, light, simple to use micro-browser using WebKit and Lua."
+HOMEPAGE="https://aidanholm.github.io/luakit/"
```

Comment:
http://luakit.org redirects to https://luakit.github.io/luakit, so I'll use that instead, I think Gentoo is trying to use more https links in homepages and such.